.dockerignore

```diff
@@ -51,9 +51,11 @@ instance/
 docs/_build/
 docs/figures/
 
-# PyBuilder
+# PyBuilder and rust
 target/
 
+lib/*/target/
```

Comment:
Good idea! I wonder if we should combine both in:

```gitignore
**/target
```

Reply:
i deliberately decided against this incase someone makes a directory named `target` in some src dir